incubator/hnc/Makefile

```diff
@@ -182,18 +182,22 @@ kind-deploy:
 	CONFIG=kind $(MAKE) deploy
 
 ###################### E2E TEST #########################
+# This test will run on the current cluster the user deployed (either kind or 
+# a kubernetes cluster). Please make sure to have the test cluster ready 
+# before running the test.
+
 # Sleep time is added for the webhook to get ready before test. The time 
 # comes from watching the log as well as multiple experiments. 
 .PHONY: e2e-test
-e2e-test: deploy
+e2e-test: 
 	go clean -testcache
 	sleep 15
 	go test ./e2e/...
 
 # The sleep time needed for kind cluster is longer than that for a real 
 # cluster, from the experiments. I don't know why - @GinnyJI, July 2020
 .PHONY: local-e2e-test
-local-e2e-test: kind-deploy
+local-e2e-test: 
```

Comment:
I'm not sure if we need this separate target now? Just have `e2e-test`?

Reply:
Done